test/unit/component/parser/json/BpmnJsonParser.flowNode.test.ts

```diff
@@ -395,7 +395,7 @@ describe.each([
       ${'incoming'} | ${'association'}  | ${'bpmnElementIncomingIds'}
       ${'outgoing'} | ${'association'}  | ${'bpmnElementOutgoingIds'}
     `(
-      `should convert as Shape with $title attribute calculated from $flowKind`,
+      `should convert as Shape without $title attribute calculated from $flowKind`,
```

Comment:
Rename into `should convert as Shape with $title attribute only calculated from $flowKind`